<?xml version="1.0"?>
<launch>

  <group ns="hauler_1">
    <node name="state_machine" pkg="state_machine" type="state_machine_rd2_node" output="screen"/>
  </group>

</launch>
